robotino_bringup/robots/kcl-hand/config/hand.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-# Vienna hand config parameters 
+# Innsbruck hand config parameters 
```

Comment:
is this the  config for the Innsbruck hand? if yes, please move the file to robotino_bringup/robots/uibk-robotino2/config/

Reply:
just wrong name. should be KCL hand. already fix.